actioncable/README.md

```diff
@@ -443,9 +443,9 @@ The Ruby side of things is built on top of [faye-websocket](https://github.com/f
 
 ## Deployment
 
-Action Cable is powered by a combination of EventMachine and threads. The
+Action Cable is powered by a combination of websocket-driver and threads. The
 framework plumbing needed for connection handling is handled in the
-EventMachine loop, but the actual channel, user-specified, work is handled
+websocket-driver loop, but the actual channel, user-specified, work is handled
```

Comment:
@claudiob what @matthewd said - it's now just threads so no need for the but clause